README.md

```diff
@@ -15,34 +18,161 @@ dependencies:
 
 `import 'package:flutter_periculum/flutter_periculum.dart';`
 
-## Example
-...dart
+## Generate Affordability 
+
+```dart
+AffordabilityResponse response; //returns an AffordabilityResponse object 
+try {
+  response = await FlutterPericulum.affordabilityAnalysis(
+      statementKey: 3,
+      dti: 2.0,
+      loanTenure: 30,
+      averageMonthlyTotalExpenses: 4000, //optional
+      averageMonthlyLoanRepaymentAmount: 1000, //optional
+      token: "..."
+      );
+} on PlatformException {
+  response = 'Failed to get platform version.';
+}
+if (!mounted) return;
+```
+## Generate Credit Score
+
+###### 
+| Parameter | Type     | Description                |
+| :-------- | :------- | :------------------------- |
+| `token` | `String` | **required** |
+| `statementKey` | `String` | **required** |
+
+
+```dart
+CreditScoreResponse response; //returns an CreditScoreResponse object 
+
+try {
+  response =
+    await FlutterPericulum.generateCreditScore(
+    token: "${dotenv.env['tokenKey']}",
```

Comment:
I would prefer if the environment did not have the access token. We don't want customers to think putting the token as an environment variable is the right thing to do. We want them to obtain the access token from their own backend.

Reply:
need more context